lib/resty/upstream/healthcheck.lua

```diff
@@ -701,4 +701,57 @@ function _M.status_page()
     return concat(bits)
 end
 
+function _M.status_page_one()
```

Comment:
Yes, I mean `status_page(upstream_name)` where the parameter `upstream_name` is optional.